test/PreAMMBatcher.e2e.ts

```diff
@@ -2,8 +2,8 @@ import { use, expect } from "chai";
 import { Contract, Wallet } from "ethers";
 import { deployContract, MockProvider, solidity } from "ethereum-waffle";
 import PreAMMBatcher from "../build/PreAMMBatcher.json";
-import UniswapV2Pair from "../node_modules/@uniswap/v2-core/build/UniswapV2Pair.json";
-import UniswapV2Factory from "../node_modules/@uniswap/v2-core/build/UniswapV2Factory.json";
+import UniswapV2Pair from "@uniswap/v2-core/build/UniswapV2Pair";
+import UniswapV2Factory from "@uniswap/v2-core/build/UniswapV2Factory";
```

Comment:
```suggestion
import UniswapV2Pair from "@uniswap/v2-core/build/UniswapV2Pair.json";
import UniswapV2Factory from "@uniswap/v2-core/build/UniswapV2Factory.json";
```

I would leave the JSON extension just to its not confused with a TS or JS module.

Reply:
Yea. I think my IDE keeps changing these without me wanting it.  Doesn't even compile without the `json`. I don't know what it keeps happening here.